CreeDictionary/DatabaseManager/cree_inflection_generator.py

```diff
@@ -44,11 +44,6 @@ def expand_inflections(
-    True
-    >>> expand_inflections(['tastawayakap+Ipc'], verbose=False)
-    {'tastawayakap+Ipc': [('tastawayakap+Ipc', {'tastawayakap'})]}
```

Comment:
These are moved to a dedicated test function. The first test is too dependent on the FST output and the newst fst fails the test. I have changed it to test some specific containment instead of strict equal.

Reply:
👍 Yeah, these tests were _way_ too fragile.